jmeter.md

```diff
@@ -0,0 +1,20 @@
+# jmeter
+
+Open source java application designed for load testing functional behavior and measure performance.
+More information: <https://jmeter.apache.org/>.
+
+- Run jmeter in cli mode for a test plan:
```

Comment:
Hi @andre-santos-fd, welcome and thanks for your submission. All these examples are in CLI mode?

Reply:
Hi @navarroaxel, thank you! Yes all the examples are in CLI because of the `-n` flag.